<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbulEduNetworkProxySettings</class>
 <widget class="QWidget" name="AbulEduNetworkProxySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configuration du proxy</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblTitre">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Configuration du serveur proxy pour accéder à Internet</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="rbProxyDesactive">
     <property name="text">
      <string>Pas de proxy</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="rbProxyAutomatique">
     <property name="text">
      <string>Détection automatique des paramètres de proxy</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="5">
    <widget class="QFrame" name="frameProxyManuel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblProxyHTTP">
        <property name="text">
         <string>Proxy HTTP :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="leProxyHTTP"/>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="lblPort1">
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="sbPort1">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>65000</number>
        </property>
        <property name="value">
         <number>3128</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QCheckBox" name="cbUtiliserPourTousProtocoles">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Utiliser ce serveur pour tous les protocoles</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblProxySSL">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Proxy SSL :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="leProxySSL">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="lblPort2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSpinBox" name="sbPort2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>65000</number>
        </property>
        <property name="value">
         <number>3128</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblProxyFTP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Proxy FTP :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="leProxyFTP">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="lblPort3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QSpinBox" name="sbPort3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>65000</number>
        </property>
        <property name="value">
         <number>3128</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblHoteSOCKS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Hôte SOCKS :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="leProxySOCKS">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="lblPort4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QSpinBox" name="sbPort4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>65000</number>
        </property>
        <property name="value">
         <number>3128</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="sbSOCKS4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>SOCKS v4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QRadioButton" name="rbSOCKS5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>SOCKS v5</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="lblPasDeProxyPour">
        <property name="text">
         <string>Pas de proxy pour :</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="5">
       <widget class="QTextEdit" name="tePasDeProxy">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(233, 234, 234);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QFrame" name="frameProxyAutoURI">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLineEdit" name="leProxyAutoURI"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnActualiser">
        <property name="text">
         <string>Actualiser</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="rbProxyManuel">
     <property name="text">
      <string>Configuration manuelle du proxy</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="rbProxySysteme">
     <property name="text">
      <string>Utiliser la configuration du système d'exploitation</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="rbProxyAutoURI">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Adresse de configuration automatique du proxy</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leProxyHTTP</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leProxySSL</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leProxyHTTP</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leProxyFTP</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leProxyHTTP</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leProxySOCKS</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbPort1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbPort2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbPort1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbPort3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbPort1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbPort4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
